tasks/apt.yml

```diff
@@ -41,6 +41,12 @@
     - python3-dev
     - python-virtualenv
 
+- name: Install python3-pyparsing
+  when: ansible_distribution == "Debian"
```

Comment:
Not sure, could you find out? You could use the debug module of ansible to print the value of `ansible_distribution_release`.